docs/example-react-redux.md

```diff
@@ -33,14 +31,18 @@ class Counter extends React.Component {
   }
 }
 
-// normally this would be:
-// export default connect(state => ({count: state.count}))(Counter)
-// but for this test we'll give it a variable name
-// because we're doing this all in one file
-const ConnectedCounter = connect(state => ({ count: state.count }))(Counter)
+export default connect(state => ({count: state.count}))(Counter)
+```
+
+For this example, we'll have a simple reducer that tracks and updates `count`:
+
+```
```

Comment:
Let's specify a language for this codefence for syntax highlighting

````
```js
````

Reply:
Good catch.